sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -0,0 +1,299 @@
+------ DESKTOP SPONSORED TILES
+WITH newtab_unnested AS (
+  SELECT AS STRUCT
+    t.client_info.client_id,
+    date(t.submission_timestamp) AS submission_date,
+    t.sample_id,
+    s.name,
+    s.category,
+    s.extra
+  FROM
+    `mozdata.firefox_desktop.newtab` t
+  CROSS JOIN
+    UNNEST(t.events) s
+  WHERE
+    date(t.submission_timestamp) = @submission_date
+),
+desktop_events_1 AS (
+  -- desktop tiles clicks and impressions
+  SELECT
+    client_id,
+    submission_date,
+    COUNTIF(
+      name = "click"
+      AND category = "topsites"
+      AND mozfun.map.get_key(extra, "is_sponsored") = "true"
+    ) AS sponsored_tiles_click_count,
+    COUNTIF(
+      name = "impression"
+      AND category = "topsites"
+      AND mozfun.map.get_key(extra, "is_sponsored") = "true"
+    ) AS sponsored_tiles_impression_count,
+  FROM
+    newtab_unnested
+  WHERE
+    submission_date = @submission_date
+  GROUP BY
+    1,
+    2
+),
+desktop_events_2 AS (
+-- desktop Sponsored Tile Dismissals and Disables
+  SELECT
+    client_id,
+    DATE(submission_timestamp) AS submission_date,
+    COUNTIF(
+      event = 'BLOCK'
+      AND value LIKE '%spoc%'
+      AND value LIKE '%card_type%'
+    ) AS sponsored_tiles_dismissal_count,
+    COUNTIF(
+      event = 'PREF_CHANGED'
+      AND source = 'SPONSORED_TOP_SITES'
```

Comment:
There is this `status` key in the `value` string indicating if it's toggled on (`"status":true`) or off (`"status":false`). So to capture disables, we can use this filter:

```
AND value LIKE '%false%'
```

FWIW, the `value` field is a JSON encoded string, so you can also use bigquery's JSON function to extract certain keys from it rather than match string patterns.